tests/lib/snaps/test-snapd-remodel-pc-24/meta/gadget.yaml

```diff
@@ -3,17 +3,6 @@ volumes:
     # bootloader configuration is shipped and managed by snapd
     bootloader: grub
     structure:
-      - name: mbr
```

Comment:
We still have this one on uc24 for amd64. Not sure why it is removed here.

Reply:
I found this
error: cannot pack "test-snapd-remodel-pc": structure #0 ("mbr"): content "mbr.img": stat test-snapd-remodel-pc/mbr.img: no such file or directory

Also discussed that with @alfonsosanchezbeato 
Perhaps we could sync to define which should be the final solution

Reply:
any progress on this?

Reply:
You could create the file from /dev/zero using 440 byte. It does not have to be functional.

Reply:
I've discussed with @valentindavid, and just to be clear, we do not need the mbr for the tests, it is just nice to have so the layout is the same as the 24 pc gadget one. So putting some 0s here will be fine - mbr is used just to print an error in case somebody tries to start a gpt UC image using the mbr boot sector as starting point instead of having UEFI.

Reply:
added the mbr.img files to the snaps and updated the gadget.yaml files